src/pages/HomePage.tsx

```diff
@@ -141,13 +149,10 @@ function HomePage() {
             src={dashboardimage}
             alt="dashboard"
           />
-          <iframe
-            className="prev-vid"
-            src="https://www.youtube.com/embed/T33NN_pPeNI"
```

Comment:
i checked the vid out and liked and subscribed

Reply:
haha funny, i was using it as placeholder until i created my own vid